verify-merge.py

```diff
@@ -10,19 +10,24 @@
 def verify():
     global args, workdir
     os.chdir('gitian-pubkeys')
-    print('Importing pubkeys...')
-    keys = [f for f in glob.glob("*.asc", recursive=True)]
-    for key in keys:
-        subprocess.check_call([GPG, '--import', key])
-    print('Refreshing pubkeys...')
-    subprocess.check_call([GPG, '--refresh'])
+    if args.import_keys:
+        print('Importing pubkeys...')
+        keys = [f for f in glob.glob("*.asc", recursive=True)]
+        for key in keys:
+            subprocess.check_call([GPG, '--import', key])
+    if args.refresh_keys:
+        print('Refreshing pubkeys...')
+        subprocess.check_call([GPG, '--refresh'])
     os.chdir('../../gitian-builder')
```

Comment:
Would be nice to clone `gitian-builder` and `monero` (if needed) so this can be run without external dependencies being assumed.  Here is a patch I tested:

Replace line 21:
```python
    os.chdir('../../gitian-builder')
```
with:
```python
    os.chdir('../../')
    if not os.path.isdir('gitian-builder'):
        subprocess.check_call(['git', 'clone', 'https://github.com/devrandom/gitian-builder.git'])
    if not os.path.isdir('monero'):
        subprocess.check_call(['git', 'clone', 'https://github.com/monero-project/monero.git'])
    os.chdir('gitian-builder')
```

Reply:
Since we already made some efforts now to be non-intrusive to the users environment, I don't want it to clone things if something in the environment is not correct. I pushed a changed that instructs the user to clone the repositories manually instead.

Reply:
Yeah, I guess that is more consistent.  My thinking was that it can't _hurt_ anything in their environment (other than using a little more disk space).  As you like though.